### PR TITLE
Add retry export endpoint

### PR DESCRIPTION
**Pull request by Cooryd**

### CODE REVIEW

File: src/domain/submission/services/submission-service.ts [hunk 1]
```diff
@@ -126,6 +126,27 @@ export class SubmissionService {
             });
         return this.get(id);
     }
+
+    async resubmit(submission: Submission): Promise<void> {
+        try {
+            await this.runMecaExport(submission, '0.0.0.0');
+            submission.status = SubmissionStatus.MECA_EXPORT_SUCCEEDED;
+        } catch (e) {
+            logger.error('Unable to resubmit', e);
```
Comment (NuclearRedeye) [inline src/domain/submission/services/submission-service.ts:135]: I assume you don't set the status here to failed as it's already in the failed state?

**Review by NuclearRedeye — APPROVED** [review]
Not taken the time to test locally myself, but have scanned the code and this looks good to me.

File: src/domain/submission/services/submission-service.ts (hunk 1 above)
Comment (NuclearRedeye) [reply src/domain/submission/services/submission-service.ts:135]: Yup, seems to be the case.

Comment (Cooryd) [reply src/domain/submission/services/submission-service.ts:135]: Correct 👍

**Pull request merged**